package.json

```diff
@@ -14,36 +11,87 @@
   "bin": "./bin/cli.js",
   "repository": {
     "type": "git",
-    "url": "git@github.com:asset-pipe/asset-pipe-cli.git"
+    "url": "https://github.com/asset-pipe/asset-pipe-client.git"
```

Comment:
hmm. Why not just `"repository": "asset-pipe/asset-pipe-client"`?

Reply:
Automated change from "boring" stuff.
Are there any advantages to `"repository": "asset-pipe/asset-pipe-client"`?

Reply:
you can change `"repository"` to a string instead of an object, and you can remove `"bugs"` entirely